CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 - Follow-up message support for application templates.
+- ``tt list`` command. It prints a list of enabled applications.
```

Comment:
`tt list`` command to print a list of enabled applications.

Reply:
Changed